.gitignore

```diff
@@ -2,10 +2,11 @@
 .DS_Store
 
 # tofu related
-tofu/main/*/.terraform
-tofu/main/*/.terraform.lock.hcl
-tofu/main/*/terraform.tfstate
-tofu/main/*/terraform.tfstate.*
+.terraform
+*terraform.tfstate.*
+.terraform.lock.hcl
+terraform.tfstate
+terraform.tfstate.*
```

Comment:
nit: are these both still required with the addition of `*terraform.tfstate.*` that eixsts on line 6?